tests/.pest/snapshots/Countries/BeninTest/it_can_calculate_benin_holidays.snap

```diff
@@ -0,0 +1,54 @@
+[
+    {
+        "name": "F\u00eate du Ramadan",
```

Comment:
Results seem to be wrong? Have you checked all results in the snapshot?

Reply:
The content of the snapshot is badly encoded for French characters (é,è,ê...), I don't know how to fix it.